scanners/sslyze.py

```diff
@@ -345,16 +351,17 @@ def init_sslyze(hostname, options, sync=False):
         return None, None
 
     if sync:
-        scanner = SynchronousScanner()
+        scanner = SynchronousScanner(network_timeout=network_timeout)
     else:
-        scanner = ConcurrentScanner()
+        scanner = ConcurrentScanner(network_timeout=network_timeout)
```

Comment:
I passed the `network_timeout` to both scanners. That said, I didn't notice any impact on improving scan times. :/